libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -223,7 +224,8 @@ export class CalendarComponent implements OnInit, OnChanges {
   constructor(
     @Inject(LOCALE_ID) locale: string,
     private elementRef: ElementRef,
-    private cdr: ChangeDetectorRef
+    private cdr: ChangeDetectorRef,
+    public translations: TranslationService
```

Comment:
Should we make getters in each component to avoid making the service public here?